lib/exchange/adapters/event_bus.ex

```diff
@@ -19,32 +19,33 @@ defmodule EventBus do
 
   def remove_listener(key) do
     if Enum.member?(@events, key) do
-      Registry.unregister(EventBus.Registry, key)
+      Registry.unregister(Regitryhange.Adapters.EventBus.Registry, key)
     else
       :error
     end
   end
 
   def cast_event(:order_cancelled, payload),
-    do: dispatch_event(:order_cancelled, %EventBus.OrderCancelled{order: payload})
+    do:
+      dispatch_event(:order_cancelled, %Exchange.Adapters.EventBus.OrderCancelled{order: payload})
 
   def cast_event(:trade_executed, payload),
-    do: dispatch_event(:trade_executed, %EventBus.TradeExecuted{trade: payload})
+    do: dispatch_event(:trade_executed, %Exchange.Adapters.EventBus.TradeExecuted{trade: payload})
 
   def cast_event(:order_expired, payload),
-    do: dispatch_event(:order_expired, %EventBus.OrderExpired{order: payload})
+    do: dispatch_event(:order_expired, %Exchange.Adapters.EventBus.OrderExpired{order: payload})
 
   def cast_event(:order_placed, payload),
-    do: dispatch_event(:order_placed, %EventBus.OrderPlaced{} = payload)
+    do: dispatch_event(:order_placed, %Exchange.Adapters.EventBus.OrderPlaced{} = payload)
 
   def cast_event(:order_queued, payload),
-    do: dispatch_event(:order_queued, %EventBus.OrderQueued{order: payload})
+    do: dispatch_event(:order_queued, %Exchange.Adapters.EventBus.OrderQueued{order: payload})
```

Comment:
maybe an `alias Exchange.Adapters.EventBus` would be good here, so these can be replaced by `EventBus.OrderCancelled`, etc